app/models/picture.rb

```diff
@@ -0,0 +1,8 @@
+class Picture < ActiveRecord::Base
+  has_attached_file :image, styles: { medium: "600x600>", thumb: "100x100>" }, default_url: "/images/:style/missing.png"
+  validates_attachment_content_type :image, content_type: /\Aimage\/.*\Z/
+  has_many :comments, -> { extending WithUserAssociationExtension }, dependent: :destroy
```

Comment:
Metrics/LineLength: Line is too long. [88/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)